tests/test_eatwell101.py

```diff
@@ -0,0 +1,83 @@
+# mypy: allow-untyped-defs
+
+from recipe_scrapers.eatwell101 import EatWell101
+from tests import ScraperTest
+
+
+class TestEatWell101Scraper(ScraperTest):
+
+    scraper_class = EatWell101
+
+    def test_host(self):
+        self.assertEqual("eatwell101.com", self.harvester_class.host())
+
+    def test_author(self):
+        self.assertEqual("Christina Cherrier", self.harvester_class.author())
+
+    def test_title(self):
+        self.assertEqual(
+            "Creamy Spinach Turkey Meatballs", self.harvester_class.title()
+        )
+
+    def test_category(self):
+        self.assertEqual(
+            "Chicken, Cook, Cooking &amp; Meals, main dish recipes, Poultry recipes, ",
+            self.harvester_class.category(),
+        )
+
+    def test_total_time(self):
+        self.assertEqual(40, self.harvester_class.total_time())
+
+    def test_yields(self):
+        self.assertEqual("4 servings", self.harvester_class.yields())
+
+    def test_image(self):
+        self.assertEqual(
+            "https://www.eatwell101.com/wp-content/uploads/2021/05/Creamy-Turkey-Meatballs-Recipe.jpg",
+            self.harvester_class.image(),
+        )
+
+    def test_ingredients(self):
+        expected_ingredients = [
+            "1/2 lb (220g) ground turkey meat",
+            "1/2 lb (220g) ground chicken meat",
+            "1/2 cup shredded mozzarella (or cheddar, provolone…)",
+            "4 cloves garlic, grated + 4 cloves garlic, minced",
+            "1 teaspoon Italian seasoning",
+            "1/2 teaspoon red crushed chili pepper flakes, optional",
+            "1 crumbled bouillon cube, optional",
+            "Salt and fresh cracked black pepper, to taste",
+            "1 cup fresh chopped cilantro (or parsley), divided",
+            "2 teaspoons olive oil",
+            "2 tablespoons butter",
+            "1 small yellow onion, diced",
+            "1/3 cup (80ml) vegetable broth",
+            "5 ounces (150g) jarred sun-dried tomato in oil, drained of oil",
+            "1 3/4 cups heavy cream",
+            "Salt and pepper, to taste",
+            "3 cups baby spinach leaves",
+            "1/2 cup grated Parmesan",
+            "1 tablespoon fresh parsley, chopped",
+        ]
+
+        self.assertEqual(expected_ingredients, self.harvester_class.ingredients())
+
+    def test_instructions(self):
+        expected_instructions = (
+            "1. To make the creamy spinach turkey meatballs: In a large bowl, combine ground turkey and ground chicken, cheese, grated garlic, Italian seasoning, bouillon cube, red chili pepper flakes, chopped cilantro, and black pepper. Mix well with your hands or fork and form medium balls. Arrange the turkey meatballs on a plate and set them aside.\n"
```

Comment:
I think I generally still prefer the `"\n".join([instruction, instruction, ...])` pattern, but after seeing it more often I'm beginning to come around to this representation despite the `\n` at the end of each line in the code.

(no change requested here, just explaining some of my thoughts as I review)

Reply:
I think this way makes it easier to read & lets the joining be handled directly in the scraper class. If you prefer the other way I can definitely try to swap it